packages/toolkit/package.json

```diff
@@ -162,6 +162,13 @@
                     "description": "%AWS.cwl.limit.desc%",
                     "maximum": 10000
                 },
+                "aws.cwl.liveTailMaxEvents": {
```

Comment:
why is this new setting needed instead of the existing `aws.cwl.limit` setting?

Reply:
The reason I added a new one is the behavior of the setting. The existing limit preference doesn't mention LiveTail nor its line-trimming behavior. 

I figured that a user may only want to keep, say, 1000 log events in their LiveTail session since its streaming and they're getting constantly able to get new lines. Though, for our Search LogGroup/View LogStream commands, they could configure a higher limit since new events won't stream in and they want more results from their request without needing to send another. 

If Toolkit prefers to not have separate preferences then I can remove.

Reply:
> The existing limit preference doesn't mention LiveTail nor its line-trimming behavior.

We can, and should, update its meaning and description.

> If Toolkit prefers to not have separate preferences then I can remove.

Yes. New settings should be avoided unless there is a strong reason to add them.